modules/engage-paella-player-7/.babelrc

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Do you want to add a comment why this empty file is included?

Reply:
This file is included because `eslint`. Try to remove the file and run `npm run eslint`
```
0:0  error  Parsing error: No Babel config file detected for /opencast/modules/engage-paella-player-7/src/index.js. Either disable config file checking with requireConfigFile: false, or configure Babel so that it can find the config files
```

I can add the `requireConfigFile: false` to the `.eslintrc.js` file, so this file is not needed. Or add `babel: {}` to the `package.json` file.

Reply:
I removed the .babelrc file